docs/Installation/topics/Setting_Up_a_Cluster.rst~

```diff
@@ -0,0 +1,113 @@
+Setting Up a Cluster
```

Comment:
this file can be removed? based on the `~` at the end, it seems to be a backup file.

Reply:
deleting